CHANGELOG.md

```diff
@@ -7,9 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
+### [0.4.0] - 2024-09-11
```

Comment:
```suggestion
### [0.5.0] - 2024-09-11
```

Reply:
Might want to add a line for `### Skipped 0.4.0` somewhere here too?

Reply:
sure